src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -173,6 +179,12 @@ public async Task<WorkerStatus> GetWorkerStatusAsync()
                 }
             }
 
+            if (_concurrencyOptions.Value.Enabled)
+            {
+                workerStatus.IsReady = IsChannelReadyForInvocations();
+                workerStatus.WorkerStats = _monitor.GetStats();
```

Comment:
Why do we need this separate WorkerChannelMonitor and WorkerStats type? It seems to me like it'd be cleaner to just incorporate the latency history samples as part of WorkerStatus. That's what it was designed for and already has some latency info. GrpcWorkerChannel can just include the timer and sample the channel periodically.